scripts/fluentui-publish/index.ts

```diff
@@ -0,0 +1,192 @@
+        npmTagForCanary,
+        '--no-verify-access',
+      ];
+      break;
+    default:
+      throw new Error(`lerna publish gets invalid bump type ${bumpType}`);
+  }
+
+  if (skipConfirm) {
+    lernaPublishArgs.push('--yes');
+  }
+
+  console.log(`Running this command: yarn ${lernaPublishArgs.join(' ')}`);
+
+  const result = spawnSync('yarn', lernaPublishArgs, {
+    cwd: fluentRoot,
+    shell: true,
+    stdio: 'inherit',
+  });
+
+  if (bumpType === 'canary') {
+    // in canary release lerna doesn't push the version change in lerna.json to remote
+    execCommandSync(gitRoot, 'git', ['add', `packages/fluentui/lerna.json`]);
+    execCommandSync(gitRoot, 'git', [
+      'commit',
+      '-m',
+      `"chore: update lerna.json after react-northstar canary release"`,
+    ]);
+    execCommandSync(gitRoot, 'git', ['push']);
+  }
+
+  if (result.status) {
+    throw new Error(result.error?.stack || `lerna publish failed with status ${result.status}`);
+  }
+}
+
+const execCommandSync = (cwd: string, command: string, args: string[]) => {
+  const result = spawnSync(command, args, {
+    cwd,
+    shell: true,
+    stdio: 'pipe',
+    encoding: 'utf-8',
+  });
+  if (result.status) {
+    throw new Error(result.error?.stack || `'${command} ${args.join(' ')}' failed with status ${result.status}`);
+  }
+  return result.stdout;
+};
+
+export function fluentuiPostPublishValidation() {
+  const gitRoot = findGitRoot();
+
+  const branch = execCommandSync(gitRoot, 'git', ['branch', '--show-current']);
+  execCommandSync(gitRoot, 'git', ['fetch', 'origin']);
+  execCommandSync(gitRoot, 'git', ['reset', '--hard', `origin/${branch}`]); // sometimes lerna add gitHead in package.json after release
+
+  // sync fluent version
+  execCommandSync(gitRoot, 'yarn', ['syncpack:fix']);
+  const gitStatus = execCommandSync(gitRoot, 'git', ['status', '--porcelain']);
+  if (gitStatus.length === 0) {
+    execCommandSync(gitRoot, 'git', ['add', `\*package.json`]);
+    execCommandSync(gitRoot, 'git', ['commit', '-m', `"chore: fix dependencies after react-northstar release"`]);
+    execCommandSync(gitRoot, 'git', ['push']);
+  }
+
+  // make sure there's no more than one fluent versions
+  execCommandSync(gitRoot, 'yarn', ['syncpack:list']);
+}
+
+// pack all public fluent ui packages, used by ci to store nightly built artifacts
+export function packFluentTarballs() {
+  const gitRoot = findGitRoot();
+  const fluentRoot = path.resolve(findGitRoot(), 'packages', 'fluentui');
+
+  const TODAY = new Date().toISOString().split('T')[0]; // yyyy-mm-dd
+
+  const fluentPackages = JSON.parse(
+    execCommandSync(fluentRoot, '../../node_modules/.bin/lerna', ['ls', '--json']).toString(),
+  );
+  const fluentPackagesNames = fluentPackages.map(pkg => pkg.name);
+
+  const replaceDepVersionWithNightlyUrl = packageLocation => {
+    const packageJson = require(`${packageLocation}/package.json`);
+    const dependencies = packageJson.dependencies || {};
```

Comment:
It will also nice to update `version` field to `0.0.0-nightly` as after exporting to CodeSandbox it looks weird:

![image](https://user-images.githubusercontent.com/14183168/107563965-007e7300-6be2-11eb-824b-77e38430e77b.png)

Reply:
now it's 
![image](https://user-images.githubusercontent.com/28751745/107596281-85d04a80-6c17-11eb-8fd0-4e9930f5b83a.png)
:)